test/per_file_coverage.sh

```diff
@@ -4,7 +4,7 @@
 # for existing directories with low coverage.
 declare -a KNOWN_LOW_COVERAGE=(
 "source/common:96.0"
-"source/common/api:82.5"
+"source/common/api:82.4"
```

Comment:
Is this change needed? I suspect it might be an artifact of a PR which landed yesterday and was reverted. Since your PR doesn't touch source/common/api, I don't *think* it need to adjust coverage. (Maybe this PR just needs a main merge?)